src/test/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcherTest.java

```diff
@@ -0,0 +1,72 @@
+package org.kiwiproject.dropwizard.util.metrics;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.kiwiproject.base.process.ProcessHelper;
+
+import java.io.ByteArrayInputStream;
+
+@DisplayName("ServerLoadFetcher")
+class ServerLoadFetcherTest {
+
+    private ServerLoadFetcher serverLoad;
+    private ProcessHelper processes;
+
+    @BeforeEach
+    void setUp() {
+        processes = mock(ProcessHelper.class);
+        serverLoad = new ServerLoadFetcher(processes);
+    }
+
+    @Test
+    void testGet_UsingActualProcess() {
+        String load = new ServerLoadFetcher().get().orElse(null);
+        assertThat(load)
+                .isNotNull()
+                .matches("^\\d+.\\d+[,]? \\d+.\\d+[,]? \\d+.\\d+$");
+    }
+
+    @Test
+    void testGet_UsingMockProcess() throws InterruptedException {
+        String output = " 18:29:28 up 34 days, 18:25, 1 user, load average: 0.88, 0.98, 1.03";
```

Comment:
You can `var` this...is there supposed to be a leading space?

Reply:
Ok, I don't think it matters, but I can remove it